ci_scripts/install.sh

```diff
@@ -25,9 +25,10 @@ popd
 # Configure the conda environment and put it in the path using the
 # provided versions
 conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
-   numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION cython=$CYTHON_VERSION matplotlib scikit-learn nbconvert nbformat jupyter_client ipython jupyter notebook ipykernel pandas
+   numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION cython=$CYTHON_VERSION matplotlib nbconvert nbformat jupyter_client ipython jupyter notebook ipykernel pandas
```

Comment:
Yes, to get 0.18, but I forgot to undo this after your release. I didn't want to develop against a version from which I new it would be outdated by the time this gets ready;) I reverted the changes.